R/ml_lda.R

```diff
@@ -7,16 +7,35 @@
 #' @param k The number of topics to estimate.
 #' @template roxlate-ml-options
 #' @template roxlate-ml-dots
+#' @param alpha Concentration parameter for the prior placed on documents' distributions over topics. This is a singleton which is replicated to a vector of length \code{k} in fitting (as currently EM optimizer only supports symmetric distributions, so all values in the vector should be the same). For Expectation-Maximization optimizer values should be > 1.0.
+#' By default \code{alpha = (50 / k) + 1}, where \code{50/k} is common in LDA libraries and +1 follows from Asuncion et al. (2009), who recommend a +1 adjustment for EM.
+#' @param beta Concentration parameter for the prior placed on topics' distributions over terms. For Expectation-Maximization optimizer value should be > 1.0 and by default \code{beta = 0.1 + 1}, where 0.1 gives a small amount of smoothing and +1 follows Asuncion et al. (2009), who recommend a +1 adjustment for EM.
+#'
+#' @references
+#' Original LDA paper (journal version): Blei, Ng, and Jordan. "Latent Dirichlet Allocation." JMLR, 2003.
+#'
+#' Asuncion et al. (2009)
+#'
 #'
 #' @family Spark ML routines
 #'
+#'
+#' @note
+#' The topics' distributions over terms are called "beta" in the original LDA paper by Blei et al., but are called "phi" in many later papers such as Asuncion et al., 2009.
+#'
+#' For terminology used in LDA model see \href{https://spark.apache.org/docs/latest/api/scala/index.html#org.apache.spark.ml.clustering.LDA}{Spark LDA documentation}.
+#'
 #' @export
 ml_lda <- function(x,
                    features = dplyr::tbl_vars(x),
                    k = length(features),
-                   ml.options = ml_options(),
-                   ...)
-{
+                   alpha = (50 / k) + 1,
+                   beta = 0.1 + 1,
+                   ml.options = NULL,
```

Comment:
@kevinushey I've updated this option with https://github.com/rstudio/sparklyr/pull/186/commits/cbc0656c1c1c7fef1ae1c94081a14564c19c7f71 thanks for the suggestion